pkg/bytecode/doc.go

```diff
@@ -0,0 +1,15 @@
+// Package bytecode contains a custom bytecode compiler and
+// virtual machine.
+//
+// Initially, an Evy program is turned into a sequence of tokens by the
+// [lexer]. Then, the [parser] generates an Abstract Syntax Tree (AST)
+// from the tokens. The [Compiler] of this package walks the AST and
+// writes the instructions to custom bytecode. For more on the AST
+// please refer to the [parser] package documentation.
```

Comment:
To keep tone consistent with other docs:

 For more on the AST see to the [parser] package documentation.

(I have not been very polite in the past and avoided pleases)